README.md

```diff
@@ -31,6 +31,7 @@ To quickly learn how to run cleanlab on your own data, first check out the [quic
 | 21  | [non_iid_detection](non_iid_detection/non_iid_detection.ipynb)  | Use Datalab to detect non-IID sampling (e.g. drift) in datasets based on numeric features or embeddings. |
 | 22  | [object_detection](object_detection/README.md)  | Train Detectron2 object detection model for use with cleanlab. |
 | 23  | [semantic segmentation](segmentation/training_ResNeXt50_for_Semantic_Segmentation_on_SYNTHIA.ipynb)  | Train ResNeXt semantic segmentation model for use with cleanlab. |
+| 24 | [llm_evals_w_crowdlab](llm_evals_w_crowdlab/llm_evals_w_crowdlab.ipynb) | Uses GPT4o and CROWDLAB to evaluate language models on a dataset labeled by multiple annotators. |
```

Comment:
Let's make this notebook #12.
Right before the other multiannotator notebook.

The file name is fine, but the notebook itself should start with a title. I recommend same title as your corresponding article.
Then you can use this same title as the Table of Content description. Note we'll want to preserve the canonical SEO term as much as possible throughout title/writing: LLM Evaluation, LLM Evals

And you don't need of the special metadata for this notebook. The special metadata is only for Tutorial notebooks that appear on doc.cleanlab.ai, this is an Example not a Tutorial.

For the Example notebooks in this repo: you should have all the outputs nicely rendered. 

For tutorial notebooks (in main cleanlab repo): you should have the outputs rendered for the initial PR just to ease review, but the final merged version should have all outputs cleared. Tutorial notebooks are auto-executed during CI to render their outputs on docs.cleanlab.ai

Reply:
Also I'd reduce references to "dataset labeled by multiple annotators" and replace them with "multiple human/AI reviewers evaluating LLM outputs"

Reply:
✔️ - moved notebook
✔️ - name change - Made _almost_ the same as Medium article but left out "cutting edge" which I think is efficacious for the medium article but not for this tutorial. 
✔️ - `Then you can use this same title as the Table of Content description`
⚠️ ❓ - `For the Example notebooks in this repo: you should have all the outputs nicely rendered`. The outputs are all rendered here, but not 100% in order as I fixed some cells. Re-running is going to be annoying bc we make a bunch of OpenAI calls so I'd respectfully ask to allow current state of "nice-ish outputs but not in order" 
- ✔️ reduce references to "dataset labeled by multiple annotators" - NO-OP given title change.

Reply:
You don't need to re-execute anything, but do try to make the notebook outputs look better in Github viewer.
Here are some cells you can clear the outputs of:

<img width="1080" alt="Screen Shot 2024-07-01 at 8 42 54 PM" src="https://github.com/cleanlab/examples/assets/1390638/f6930fa3-1b45-4ccf-9d52-2ae774e270c1">

<img width="1009" alt="Screen Shot 2024-07-01 at 8 43 31 PM" src="https://github.com/cleanlab/examples/assets/1390638/4ee39512-571c-4abd-b239-0b484664c0c5">

This nit is mostly for future reference